ubiconfig/_impl/loaders/gitlab.py

```diff
@@ -35,15 +37,21 @@ def load(self, file_name):
             raise
 
         # validate input data
-        validate_config(config_dict)
+        try:
+            validate_config(config_dict)
+        except ValidationError as e:
+            LOG.error("%s FAILED schema validation:\n%s\nSkip for now", file_name, e)
```

Comment:
You could define the message in ubiconfig.utils.config_validation. Also the format is not consistent with other log messages. Can we keep this on single line?

Reply:
yes, I'll do it.